packaging/copr/subsurface.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  libssh2-devel
 BuildRequires:  libcurl-devel
 BuildRequires:  libgit2-devel
 BuildRequires:  libmtp-devel
+BuildRequires:  LibRaw-devel
```

Comment:
Does this need to declare an installation dependency on libraw?

Reply:
IIRC the way this works is that this then automagically creates the runtime dependency.
You see that none of the other libraries like `libgit2` or `libxml2` have runtime dependencies, either